frontend/sozisel-app/src/components/TemplatesList/SearchBar/SearchBar.tsx

```diff
@@ -136,12 +141,17 @@ export default function SearchBar({ onSearch }: SearchBarProps): ReactElement {
             color="primary"
             fullWidth
             className="searchBarButton"
-            onClick={() => onAdd()}
+            onClick={onAdd}
           >
             {t("components.TemplatesList.addButtonText")}
           </Button>
         </Grid>
       </Grid>
+      <Snackbar open={!!error} autoHideDuration={3000}>
```

Comment:
we should probably extract hide duration as we use it pretty often to some const variable